contracts/StreamManager.sol

```diff
@@ -238,6 +208,9 @@ contract StreamManager is IStreamManager, ReentrancyGuard {
         uint256 terminatedAt = block.timestamp;
         if (!isPayee[_payee]) revert NotPayee();
         if (streamInstances[_payee].terminatedAt != 0) revert AlreadyTerminatedOrTerminating();
+        /// Terminte in cliff period
```

Comment:
typo here

Reply:
Better here to get the storage value and modify it

`OpenStream storage streamInstance = streamInstances[_payee];`

Reply:
yes

Reply:
Fixed